src/LibVLCSharp.Forms/Shared/EqualizerUtils.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using LibVLCSharp.Shared;
+using Xamarin.Forms;
+
+namespace LibVLCSharp.Forms.Shared
```

Comment:
Most of this class, except for one function, could be located in the main libvlcsharp package. Let's do it?